matter_server/common/helpers/util.py

```diff
@@ -194,8 +202,14 @@ def parse_value(name: str, value: Any, value_type: Any, default: Any = MISSING)
         return float(value)
     if value_type is int and isinstance(value, str) and value.isnumeric():
         return int(value)
+    # handle bytes values (sent over the wire as base64 encoded strings)
     if value_type is bytes and isinstance(value, str):
-        return b64decode(value.encode())
+        try:
+            return b64decode(value.encode("utf-8"))
+        except binascii.Error:
+            # unfortunately sometimes the data is malformed
+            # as it is not super important we ignore it (for now)
+            return b""
```

Comment:
Right. I was thinking if we should be more explicit with these things. Is there a special instance we can return to say that this part of the data is known invalid?

Reply:
we could insert an enum member maybe ? like UNKNOWN ?
But I dont think we can modify the enum on the fly, maybe we should just return a special type here like a string with "UNKNOWN_VALUE" or "PARSE_ERROR" and define that as constant ?

Reply:
Ah sorry, feedback was about the bytes instead of enum but well, same answer I guess. Maybe we should define a constant that we return which we can also check in the consumer

Reply:
Yes, something like that. 👍 

This was just a side note though, so we don't need to do it here, right now.